SimPEG/Maps.py

```diff
@@ -118,6 +118,78 @@ def __mul__(self, val):
     def __str__(self):
         return "%s(%s,%s)" % (self.__class__.__name__, self.shape[0], self.shape[1])
 
+
+class IdentityMap_Meshless(IdentityMap):
```

Comment:
Can we put three extra if statements instead of a new class?

```
if type(self.mesh) is int:
    self._nP = self.mesh
    self.mesh = None
```

Could change the input to `mesh_or_nP`?